ostd/src/arch/riscv/mod.rs

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Platform-specific code for the RISC-V platform.
+
+pub mod boot;
+pub(crate) mod cpu;
+pub mod device;
+pub mod iommu;
+pub(crate) mod irq;
+pub(crate) mod mm;
+pub(crate) mod pci;
+pub mod qemu;
+pub mod serial;
+pub mod task;
+pub mod timer;
+pub mod trap;
+
+use core::sync::atomic::Ordering;
+
+#[cfg(feature = "cvm_guest")]
+pub(crate) fn check_tdx_init() {
+    // Unimplemented, no-op
```

Comment:
This method should be given an arch-independent name such as `init_cvm_guest` since there won't be TDX on RISC-V.